openedx/features/colaraz_features/helpers.py

```diff
@@ -153,27 +159,29 @@ def create_site_configurations(sites, organization, university_name, platform_na
     Returns:
         (Pair<SiteConfiguration, SiteConfiguration>): named tuple containing lms, studio sites configurations.
     """
```

Comment:
Remove this since you are not returning anything.

Reply:
Updated.